src/ply_io.cpp

```diff
@@ -334,12 +334,12 @@ bool findVertexElements(std::vector<p_ply_element>& vertexElements,
             g_logger.warning("Ignoring unrecogized ply element: %s", name);
         }
     }
-    if (positionIndex == -1)
+    /*if (positionIndex == -1)
     {
         g_logger.error("%s", "No vertex position found in ply file");
         return false;
-    }
-    if (positionIndex != 0)
+    }*/
```

Comment:
Actually isn't this already covered? It will be caught in `PointArray::loadFile`, for instance. Where do you expect it to go wrong?